Project.toml

```
@@ -1,6 +1,6 @@
 name = "GPUArrays"
 uuid = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7"
-version = "11.2.2"
+version = "11.3.0"
```

Seems like a bugfix; I don't think this needs a minor bump?
```suggestion
version = "11.2.3"
```

---

Yes that's true, good point.